plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -389,7 +389,7 @@ def summary
 
   def show_extraction_progress(count)
     progress = Machinery::pluralize(
-      count, "-> Extracted %d file or tree", "-> Extracted %d files and trees",
+      count, " -> Extracted %d file or tree", "-> Extracted %d files and trees",
```

Comment:
Why is there a space for the singular, but no space for the plural form?

There also is a trailing comma, which probably shouldn't be there.